pkgs/servers/dns/pdns/default.nix

```diff
@@ -59,16 +59,16 @@ stdenv.mkDerivation (finalAttrs: {
   ]);
 
   configureFlags = [
-    "--disable-silent-rules"
-    "--enable-dns-over-tls"
-    "--enable-unit-tests"
-    "--enable-reproducible"
-    "--enable-tools"
-    "--enable-ixfrdist"
-    "--enable-systemd"
-    "--with-libsodium"
-    "--with-sqlite3"
-    "--with-libcrypto=${openssl.dev}"
+    (lib.enableFeature false "silent-rules")
```

Comment:
Might be better clubbing them together in a set with `lib.mapAttrsToList`.

Reply:
I think the same, despite being more verbose.